rules/framework.bzl

```diff
@@ -157,6 +157,7 @@ def _get_virtual_framework_info(ctx, framework_files, compilation_context_fields
         ctx,
         hdrs = outputs.headers,
         module_map = outputs.modulemap,
+        swiftmodule_contents = outputs.swiftmodule + outputs.swiftdoc,
```

Comment:
okay so here we are looking at files (not dir right?) ending with `.swiftmodule` and `.swiftdoc`. I think there are also `.swiftinterface` and `.swiftinterface-e` which are currently not collected. They are under .swiftmodule dir right now for one .xcframework we are working with.
For now I add these two, just FYI in future we might encounter swift declaration not found error and it is probably due to not including `swift interface` here IMO.

Reply:
Can you add a comment here mentioning the potential `swift declaration not found error` error and how to proceed in case it happens?

Reply:
+1 will do